Generate SBOM for generated Package (#4771)
Scille/parsec-cloud: newsfragments/4770.doc.rst

> Add SBOM (Software Bills Of Materials) generation when packaging the softwares.
> This provide the list of dependencies used to build the software.

Reviewer: ```suggestion
Add SBOM (Software Bills Of Materials) generation on software packaging.
This provides the list of dependencies used to build the software.
```
